_pkgdown.yml

```diff
@@ -1,4 +1,10 @@
 url: https://hidef-aerial-surveying.github.io/stochLAB/
 template:
-  bootstrap: 5
+  package: rotemplate
 
+authors:
+  HiDef:
+    href: "https://www.hidefsurveying.co.uk/"
```

Comment:
btw the link at https://github.com/HiDef-Aerial-Surveying is not this one but it redirects to this.